Analytics-CSharp/Segment/Analytics/Analytics.cs

```diff
@@ -63,14 +66,13 @@ public Analytics(Configuration configuration)
         public void Process(RawEvent incomingEvent)
         {
             incomingEvent.ApplyBaseData();
-            
+
             analyticsScope.Launch(analyticsDispatcher, async () =>
```

Comment:
`async` can be removed, since we are not awaiting anything inside the function anymore.